api/applications_api.py

```diff
@@ -0,0 +1,188 @@
+from datetime import datetime
+from dateutil.parser import parse
+from django.shortcuts import get_object_or_404
+from django.utils import timezone
+from rest_framework import viewsets, serializers
+from rest_framework.response import Response
+from applications.models import (
+    Application,
+    Organisation,
+    ApplicationEvent,
+    Recurrence,
+    Address,
+    Person,
+)
+
+
+MINIMUM_TIME = timezone.datetime(
+    1970, 1, 1, 0, 0, 0, 3, timezone.get_default_timezone()
+)
+
+MAXIMUM_TIME = timezone.datetime(
+    2099, 1, 1, 0, 0, 0, 3, timezone.get_default_timezone()
+)
+
+
+class PersonSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Person
+
+        fields = "__all__"
+
+
+class PersonViewSet(viewsets.ModelViewSet):
+
+    queryset = Person.objects.all()
+
+    serializer_class = PersonSerializer
+
+    def perform_create(self, serializer):
+        serializer.save()
+
+
+class AddressSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Address
+
+        fields = "__all__"
```

Comment:
Not sure if this is work-in-progress, but note that using `__all__` will make everything in the database table readable _and_ writable in the API by default (at least in the case of the `ModelViewSet` below). This may cause problems for example if someone adds new backend-only fields and is not aware that they're automatically made public and possibly writable.

Reply:
Yeah, leftover from some quick testing, will fix.